tempesta_fw/http.c

```diff
@@ -3403,6 +3436,47 @@ tfw_cfgop_resp_body(TfwCfgSpec *cs, TfwCfgEntry *ce)
 	return tfw_http_config_resp_body(code, ce->vals[1]);
 }
 
+static int
+tfw_cfgop_whitelist_mark(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	unsigned int i;
+	const char *val;
+
+	if (!ce->val_n) {
+		TFW_ERR_NL("Invalid number of arguments: %zu\n", ce->val_n);
```

Comment:
It will always say `Invalid number of arguments: 0`. May be a message like `("%s: At least one argument is required", ce->name)` will be more user friendly?

Reply:
Corrected.